test/unit/lodash.spec.js

```diff
@@ -0,0 +1,800 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+// var sinon = require('sinon');
+
+describe('Lodash training', function() {
+    var params;
+    // ----------------------- LANG ----------------------- //
```

Comment:
Pull requests are basically a requests to merge your data into the main repository/branch. You shouldn't leave such meaningful comments(just keep them on separate branches!)